### DISCUSSION TITLE
Facial Recognition Model

### DISCUSSION THREAD

**Discussion by zodcas** [immich-app/immich#4924] [Q&A] [answered]
Is it possible to change the Facial Recognition model? I wanted to try antelopev2 instead of buffalo.

**Comment by mertalev** [comment]
Hi, we don't support that model at the moment since it isn't available through their code API. I actually don't see an official link to this model in general.

**Reply by zodcas** [reply]
Found the link for it here: https://github.com/deepinsight/insightface/tree/master/python-package

**Reply by mertalev** [reply, marked as answer]
Thanks for the link! This model will be added with #4952.
